README.md

```diff
@@ -13,8 +13,42 @@ npm install @js-soft/codemagic-tools
 
 ## Usage
 
+### Teams Messaging
+
+#### Build Messages
+```bash
+jscm teams-develop --platform <platform> --projectName <projectName>
+```
+This command will inform about a new build and additionally provide a link to the build. In case of a failed build it will also provide a link to the build log.
```

Comment:
```suggestion
This command will inform about a new development build and additionally provide a link to the build. In case of a failed build it will also provide a link to the build log.
```